Marlin/src/lcd/e3v2/common/dwin_set.h

```diff
@@ -25,7 +25,10 @@
 #define Language_English    1
 #define Language_Chinese    2
 
-#define ICON 7 // 7.ICO
+#define ICON 7 // Stock icons = 7.ICO; Custom icon DWIN_SET may be different
+#ifdef ICON
+  #warning "Check your DWIN_SET .ICO file for this value of ICON"
+#endif
```

Comment:
This is asking a lot of users. Our instructions tell users to use our provided `DWIN_SET`. I'm not sure I would even know how to respond to this message during a Marlin build.

Reply:
I suppose that can be reverted. The thing is some `DWIN_SET`s have other than **7.ICO**. I couldn't tell you why the change, basically any other custom display set uses **9.ICO** (which would mean it would need to be `#define ICON 9` instead of "7" if that user has or will install the `DWIN_SET` on their LCD).

It would make sense to have them all equal the same #.ICO since a user is only going to be using one at a time anyway.

I just know from other forums I see users having difficulty regarding this. say they want to install a custom display, the `DWIN_SET` has **9.ICO**, they go to install this firmware with `ICON 7` and the display ICON's dont show.

if that happens they either can change it through the firmware code like we see here, or they rename the .ICO file.

we can probably either change one or the other to make it easier for everyone else:  
make it so that it is...  
1. `#define ICON 9`  
1a. and change the default provided `DWIN_SET` to **9.ICO**  
2. or go into the other custom `DWIN_SET`s and change it to **7.ICO**

Actually, I would simply change it for **9.ICO** and `ICON 9` instead of 7.  
the reason why is most users already have a custom `DWIN_SET` installed on their LCD. so thats why I added a warning when compiling to check/change this value to match the one installed.

maybe this warning isnt called for, but just a comment will do.

Reply:
> The thing is some DWIN_SETs have other than 7.ICO

What we want is for users who are running mainline Marlin to use the `DWIN_SET` that we supply for mainline Marlin, which works with all the Ender-3 V2 UI options. We also provided the option in the past to use the Creality UI without updating the display, but that was the only UI giving that option. At this point that might not be an option anymore, but if that option can be retained it will ease migration for any users who just want to update the firmware without modifying their configs or picking a different UI.

Reply:
This is what used to be here, which we got rid of (not by mistake):
```cpp
//#define USE_STOCK_DWIN_SET  // Use the Creality stock DWIN_SET instead of Marlin's unified DWIN_SET by The-EG & thinkyhead
#ifdef USE_STOCK_DWIN_SET
  #define ICON 9 // 9.ICO
#else
  #define ICON 7 // 7.ICO
#endif
```

Reply:
how about then:
```y
//#define USE_STOCK_DWIN_SET  // Use the Creality stock DWIN_SET instead of Marlin's unified DWIN_SET by The-EG & thinkyhead  
#ifdef USE_STOCK_DWIN_SET  
  #define ICON 9 // 9.ICO  
#else  
  #define ICON 7 // 7.ICO  
  #warning "If you do have a different DWIN_SET installed on your display, you may need to enable USE_STOCK_DWIN_SET."  
#endif  

```

Reply:
We removed the `USE_STOCK_DWIN_SET` define because the code that supported that set has been completely removed, and users must use our `DWIN_SET` with stock Marlin.